### PR TITLE
Models

### PR DESCRIPTION
**Pull request by sarkasvo**
Recipe and Menu models added

### CODE REVIEW

File: app/models/menu.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Menu < ApplicationRecord
+  belongs_to :recipe
```
Comment (patulka) [inline app/models/menu.rb:2]: has_many :recipes, through menu_recipes

**Review by patulka — CHANGES_REQUESTED** [review]
you know

**Comment by sarkasvo** [comment]
- models Recipe, Menu and WeekMenu created
- very basic seed for 10 random recipes (to be able to use the latter)
-  very basic display of weekly menu 
Screen Shot 2021-04-26 at 17 07 50

**Review by patulka — APPROVED** [review]

**Review by patulka — APPROVED** [review]
changed schema a bit, based on DB (no user in menus, but week_menu)

**Pull request merged**